README.md

```diff
@@ -25,10 +25,15 @@ git clone --depth 1 -b "v0.6.0" --single-branch https://github.com/k8s-school/ki
 cd kind-travis-ci
 ./kind/k8s-create.sh -s
 
+# Install Qserv in two lines
+curl -fsSL https://raw.githubusercontent.com/lsst/qserv-operator/tickets\/DM-24372/deploy/qserv.sh | bash -s
```

Comment:
change DM-24372 to `master` before checkin I think?

Reply:
:+1: